src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Controls/DataGridCellInfo.cs

```diff
@@ -184,7 +184,7 @@ public override bool Equals(object obj)
 
         internal bool EqualsImpl(DataGridCellInfo cell)
         {
-            return (cell._column == _column) && (cell.Owner == Owner) && (cell._info == _info);
+            return (cell._column == _column) && (cell.Owner == Owner) && (cell.Info == Info);
```

Comment:
How does this even compile, as far as I can see there is no `Info` member in the class?

Reply:
Ughh) Sorry.. There should be Item property. I'll fix.